app/support/notification_sender.rb

```diff
@@ -74,10 +74,18 @@ def reviewed_at
   class AccountNotifier
 
     def notify_accounts(accounts_to_notify)
+      notifications = {}
       accounts_to_notify.each do |account_id, facility_id|
-        account = Account.find(account_id)
-        account.notify_users.each do |u|
-          Notifier.review_orders(user_id: u.id, facility_id: facility_id, account_id: account_id).deliver_now
+        user_ids = AccountUser.where(account_id: account_id, user_role: ["Business Administrator", "Owner"]).pluck(:user_id)
+        user_ids.each do |user_id|
+          notifications[user_id] ||= { facility_id => [] }
+          notifications[user_id][facility_id] << account_id
+        end
+      end
+
+      notifications.each do |user_id, facility_ids|
```

Comment:
This method is a whole lot more complicated than the old version. Is there something we can do further up the stack to clean this up? Is there a clearer way to write this? What if we sent one email per user instead of one email per user per facility? The new view is cross-facility as well, correct?